tests/rules/javascript/test.complex_prefs_defaults.js

```diff
@@ -0,0 +1,42 @@
+import JavaScriptScanner from 'scanners/javascript';
+import { VALIDATION_WARNING } from 'const';
+import { COMPLEX_PREFS_DEFAULTS } from 'messages/javascript';
+
+describe('complex_prefs_defaults', () => {
+  it('should pass when only calling pref', () => {
+    var code = 'pref();';
+    var jsScanner = new JavaScriptScanner(code,
+                                          'defaults/preferences/pref.js');
+
+    return jsScanner.scan()
+      .then((validationMessages) => {
+        assert.equal(validationMessages.length, 0);
+      });
+  });
+
+  it('should fail due to complex code', () => {
```

Comment:
The message is complex code, but really it's just not calling the global `pref()`; something like:

```js
it('should fail if pref() is not called as a global')
```
or something like that, as we're essentially making sure pref() as a method of anything but the global object isn't okay.